src/sourmash/sbt.py

```diff
@@ -831,8 +831,12 @@ def load(cls, location, *, leaf_loader=None, storage=None, print_version_warning
         sbt_fn = os.path.join(dirname, sbt_name)
         if not sbt_fn.endswith('.sbt.json') and tempfile is None:
             sbt_fn += '.sbt.json'
-        with open(sbt_fn) as fp:
-            jnodes = json.load(fp)
+
+        try:
+            with open(sbt_fn) as fp:
+                jnodes = json.load(fp)
+        except NotADirectoryError as exc:
```

Comment:
TIL https://docs.python.org/3/library/exceptions.html#NotADirectoryError

Still a bit weird that `open` is throwing it...